release-notes/8.0/preview/api-diff/preview2/Microsoft.WindowsDesktop.App/8.0-preview2_System.ComponentModel.Design.Serialization.md

```diff
@@ -0,0 +1,45 @@
+# System.ComponentModel.Design.Serialization
+
+``` diff
+ namespace System.ComponentModel.Design.Serialization {
+     public class DesignerSerializationManager : IDesignerSerializationManager, IServiceProvider {
+-        public DesignerSerializationManager(IServiceProvider provider);
++        public DesignerSerializationManager(IServiceProvider? provider);
+-        public IContainer Container { get; set; }
++        public IContainer? Container { get; set; }
+-        public object PropertyProvider { get; set; }
++        public object? PropertyProvider { get; set; }
+-        ContextStack System.ComponentModel.Design.Serialization.IDesignerSerializationManager.Context { get; }
```

Comment:
@carlossanlop, i remember you made changes to ignore changes in just full namespace, like this.

Reply:
Yeah, I did. Good catch. Let me investigate...